sites/fast-website/src/app/components/content-placement-container/templates/feature.template.ts

```diff
@@ -3,10 +3,18 @@ import { html } from "@microsoft/fast-element";
 const featureTemplate = html`<site-feature-card>
     <h4>${x => x.header}</h4>
     <p slot="body">${x => x.body}</p>
-    <fast-anchor slot="footer" href=${x => x.githubLink} appearance="lightweight"
+    <fast-anchor
+        slot="footer"
+        href=${x => x.githubLink}
+        appearance="lightweight"
+        aria-label=${x => x.header}
         >View Github</fast-anchor
     >
-    <fast-anchor slot="footer" href=${x => x.documentationLink} appearance="lightweight"
+    <fast-anchor
+        slot="footer"
+        href=${x => x.documentationLink}
+        appearance="lightweight"
+        aria-label=${x => x.header}
```

Comment:
@khamudom does this need an aria-label? Doesn't this already have a visual label?

Reply:
also will remove this, for now, to see if we get any feedback from CELA